src/metabase/search/impl.clj

```diff
@@ -197,7 +197,9 @@
          :collection_type
          :archived_directly
          :display_name
-         :effective_parent))))
+         :effective_parent
+         :source
```

Comment:
mildly concerned these keys could collide with other models in the future and get silently dropped here, but not sure this is a valid concern or worth restricting to transforms